runtime/gc_glue_java/MarkingSchemeRootClearer.cpp

```diff
@@ -329,6 +330,22 @@ MM_MarkingSchemeRootClearer::scanContinuationObjects(MM_EnvironmentBase *env)
 						while (NULL != object) {
 							gcEnv->_markJavaStats._continuationCandidates += 1;
 							omrobjectptr_t next = _extensions->accessBarrier->getContinuationLink(object);
+
+							{
+								/* fixup leftover from backout concurrent scavenge */
```

Comment:
be more specific.... for example: in case of CS backout make sure the list contains the new version of the object or if there is only one version, due to being self-forwarded, restore the self-forwarded bit

we could even check that CS backout was on before proceeding by this step, but probably not much savings anyway